lib/background_worker/logging_concern.rb

```diff
@@ -0,0 +1,17 @@
+require 'active_support/concern'
+
+module BackgroundWorker
+  module LoggingConcern
+    extend ActiveSupport::Concern
+    attr_reader :job_id
```

Comment:
job_id is provided by the class we include into so you don't need to redefine it here